packages/editor/src/dataviews/types.ts

```diff
@@ -31,7 +40,7 @@ export interface PostWithPageAttributesSupport extends BasePost {
 	menu_order: number;
```

Comment:
Not changed here, but surprized to see `menu_order` for templates. Is this a leftover or we use that?